src/participant_network.star

```diff
@@ -150,7 +163,12 @@ def launch_participant_network(plan, participants, network_params, global_log_le
 			fail("Unsupported launcher '{0}', need one of '{1}'".format(cl_client_type, ",".join([cl.name for cl in cl_launchers.keys()])))
 
 		cl_launcher, launch_method = cl_launchers[cl_client_type]["launcher"], cl_launchers[cl_client_type]["launch_method"]
-		cl_service_name = "cl-{0}-{1}-{2}".format(index+1, cl_client_type, el_client_type)
+		if len(participants) > 9:
+			index_str = zfill_custom(index+1, 2)
```

Comment:
can we put the numbers in constants above

Reply:
yes, done pls check